airflow/jobs/scheduler_job.py

```diff
@@ -592,7 +592,11 @@ def execute_callbacks(
                 elif isinstance(request, DagCallbackRequest):
                     self._execute_dag_callbacks(dagbag, request, session)
             except Exception:  # pylint: disable=broad-except
-                self.log.exception("Error executing callback for File: %s", request.full_filepath)
+                self.log.exception(
+                    "Error executing %s callback for file: %s",
+                    request.__class__.__name__,
+                    request.full_filepath
+                )
```

Comment:
This is unrelated change, but address one of the comments from Tomek on the Scheduler HA PR